### PR TITLE
Update run local instruction to point to READMEs

### PR DESCRIPTION
**Pull request by line47**

### CODE REVIEW

**Comment by jseppi** [comment]
Should we just get rid of this page entirely?

**Comment by wslack** [comment]
You could fold it into https://federalist-docs.18f.gov/pages/how-federalist-works/for-federalist-developers/

**Comment by line47** [comment]
I feel we could and if we wanted to link to the READMEs we could do so from where the repos are talked about - https://federalist-docs.18f.gov/pages/how-federalist-works/#functional-repositories

**Comment by wslack** [comment]
@line47 @jseppi I'm going to actually push against that. That page is getting scrutinized by people that want to buy Federalist, not run it locally. The audience that runs Federalist locally is separate.

**Comment by jseppi** [comment]
Do [potential] partners need to know about the involved Federalist repositories at all? Who is the audience for https://federalist-docs.18f.gov/pages/how-federalist-works/?

Information architecture is hard!

**Comment by wslack** [comment]
Yes, we get scrutinized by compliance people. They want to understand, not to run.

**Comment by wslack** [comment]
If our customers/partners had a need to run it locally, it would be different, but I don't think they do.

**Comment by wslack** [comment]
Let's add a pointer to ## Running locally for the builder.

File: pages/how-federalist-works/for-federalist-developers.md [hunk 1]
```diff
@@ -3,6 +3,15 @@ title: For Federalist Developers
 parent: How Federalist Works
 ---
 
+## Running Federalist locally
+See the [Federalist GitHub repo README](https://github.com/18F/federalist#running-locally-development) for instructions.
```
Comment (line47) [inline pages/how-federalist-works/for-federalist-developers.md]: Updated the repo links to anchor to sections talking about running local 👍 
@wslack

**Comment by line47** [comment]
Removed page from nav

**Pull request merged**